documentation/ntfs/ntfs_fr.md

```diff
@@ -321,19 +323,24 @@ Colonne | Type | Contrainte | Commentaire
 stop_id | chaine | Requis | Identifiant de l'arrêt
 visible | entier | Optionnel | Indique si le stop peut être retourné dans l'autocomplétion (valeur 1) ou s'il est ignoré (valeur 0).
 stop_name | chaine | Requis | Nom de l'arrêt
-stop_lat | décimal | Requis | Latitude. Ce champ est obligatoire pour les arrêts physiques (location_type = 0) même si le champ geometry_id est renseigné afin de faciliter sa lecture. Il est conseillé pour les zones d'arrêts (location_type = 1), et inutile pour les autres cas.
-stop_lon | décimal | Requis | Longitude. Ce champ est obligatoire pour les arrêts physiques (location_type = 0) même si le champ geometry_id est renseigné afin de faciliter sa lecture. Il est conseillé pour les zones d'arrêts (location_type = 1), et inutile pour les autres cas.
-fare_zone_id | chaine | Optionnel | Zone tarifaire de l'arrêt
+stop_code | chaine | Requis | Code de l'arrêt connu du voyageur. Dans le cas d'une entrée/sortie, contient le code/le numéro de l'entrée/sortie.
```

Comment:
Why stop_code is required? We were using object_codes why not use this for entrances too?
Otherwise if we have a stop_code on a station or stop_point what will the object_system be in Kraken?